chainercv/links/model/sequential_feature_extractor.py

```diff
@@ -0,0 +1,132 @@
+import collections
+
+import chainer
+
+
+class SequentialFeatureExtractor(chainer.Chain):
+
+    """A feature extractor model with a single-stream forward pass.
+
+    This class is a base class that can be used for an implementation of
+    a feature extractor model.
+    The link takes as an argument :obj:`layers` that specifies the computation
+    conducted in :meth:`__call__`.
+    :obj:`layers` is a list or :obj:`collections.OrderedDict` of
+    callable objects called layers, which are going to be called sequentially
+    starting from the top to the end.
+    A :obj:`chainer.Link` object in the sequence will be added as
+    a child link of this object.
+
+    :meth:`__call__` returns single or multiple features that are picked up
```

Comment:
How about adding a sentence after this one saying

"Note that the number of features can be one."

I think it is better to make all the related nouns plural because there will be too many `X(s)` in this paragraph.